kubernetes/cpi-inner.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
dont believe this file is necessary given that https://github.com/cloudfoundry/bosh-deployment/blob/master/bosh-dev.yml exists.